README.md

```diff
@@ -251,8 +251,6 @@ Alternatively, install Starship using any of the following package managers:
 <details>
 <summary>Windows</summary>
 
-Install the latest version for your system with the MSI-installers from the [releases section](https://github.com/starship/starship/releases/latest).
-
```

Comment:
Please keep this line, I'd still like to mention that starship can be installed without any extra tools.

Reply:
I added back these lines in my latest commit